Stripe/STPApplePayContext.m

```diff
@@ -124,6 +126,28 @@ - (void)_end {
     self.viewController = nil;
     self.delegate = nil;
 }
+
+- (nullable STPPaymentIntentShippingDetailsParams *)_shippingDetailsFromPKPayment:(PKPayment *)payment {
+    CNPostalAddress *address = payment.shippingContact.postalAddress;
+    NSPersonNameComponents *name = payment.shippingContact.name;
+    if (address.street == nil || name == nil) {
+        // The shipping address street and name are required parameters for a valid STPPaymentIntentShippingDetailsParams
+        return nil;
+    }
+   
+    STPPaymentIntentShippingDetailsAddressParams *addressParams = [[STPPaymentIntentShippingDetailsAddressParams alloc] initWithLine1:payment.shippingContact.postalAddress.street];
+    addressParams.city = address.city;
+    addressParams.state = address.state;
+    addressParams.country = address.ISOCountryCode;
+    addressParams.postalCode = address.postalCode;
+    
+    NSPersonNameComponentsFormatter *formatter = [NSPersonNameComponentsFormatter new];
+    formatter.style = NSPersonNameComponentsFormatterStyleLong;
```

Comment:
This produces a name like "Dr. Jonathan Maple Appleseed Esq." vs. `...StyleDefault` which produces a name like "Jonathan Appleseed".